<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsReport</class>
 <widget class="QDialog" name="ItemsReport">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report of Items</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTable">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Inventory</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>log_import</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>log_export</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnShowTable">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
